backend/src/main/java/com/cruru/applicant/service/ApplicantService.java

```diff
@@ -70,6 +71,19 @@ private void validateRejectable(Applicant applicant) {
         }
     }
 
+    @Transactional
+    public void unreject(long applicantId) {
+        Applicant applicant = findById(applicantId);
+        validateUnrejectable(applicant);
+        applicant.unreject();
+    }
+
+    private void validateUnrejectable(Applicant applicant) {
+        if (!applicant.isRejected()) {
```

Comment:
```suggestion
        if (applicant.isUnrejected()) {
```
Application 내에 isUnrejected 메서드 따로 두면 가독성이 좋아질 것 같은데, 어떻게 생각하시나요?